sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/JS.scala

```diff
@@ -50,10 +64,10 @@ object JS {
     JS(New(varRef("RegExp"), List(StringLiteral(value))))
 
   /** Block of several statements */
-  def block(stats: JS*): JS = JS(Block(stats.map(_.tree).to[List]))
+  def block(stats: JSLike*): JS = JS(Block(stats.map(_.tree).to[List]))
```

Comment:
I would keep `JS` here because that would not make a lot of sense to allow a (pure) `JSON` value in a list of statements.